control/controlclient/map.go

```diff
@@ -259,10 +262,39 @@ func (ms *mapSession) updateStateFromResponse(resp *tailcfg.MapResponse) {
 		ms.lastDERPMap = dm
 	}
 
+	var packetFilterChanged bool
+	// Older way, one big blog:
```

Comment:
blob

Reply:
sad to say that somebody else has already registered blobber.com